website/content/docs/secrets/key-management/index.mdx

```diff
@@ -145,19 +148,23 @@ KMS providers:
 
 - [Azure Key Vault](/docs/secrets/key-management/azurekeyvault)
 - [AWS KMS](/docs/secrets/key-management/awskms)
+- [GCP Cloud KMS](/docs/secrets/key-management/gcpkms)
 
 Refer to the provider-specific documentation for details on how to properly configure each provider.
 
 ## Compatibility
 
 The following table defines which key types are compatible with each KMS provider.
 
-| Key Type       | Azure Key Vault | AWS KMS |
-| -------------- | --------------- | ------- |
-| `aes256-gcm96` | No              | **Yes** |
-| `rsa-2048`     | **Yes**         | No      |
-| `rsa-3072`     | **Yes**         | No      |
-| `rsa-4096`     | **Yes**         | No      |
+| Key Type       | Azure Key Vault | AWS KMS | GCP Cloud KMS |
+| -------------- | --------------- | ------- | ------------- |
+| `aes256-gcm96` | No              | **Yes** | **Yes**       |
+| `rsa-2048`     | **Yes**         | No      | **Yes**       |
+| `rsa-3072`     | **Yes**         | No      | **Yes**       |
+| `rsa-4096`     | **Yes**         | No      | **Yes**       |
+| `ecdsa-p256`   | No              | No      | **Yes**       |
+| `ecdsa-p384`   | No              | No      | **Yes**       |
+| `ecdsa-p521`   | No              | No      | No            |
```

Comment:
It seems odd that you can create this key type, but that it can't really be used for any of the cloud KMS'es. Maybe that's fine though.

Reply:
Ya.. It was too late before I found out it wasn't supported in GCP KMS. Good news is that it'll be supported by Azure (see [docs](https://docs.microsoft.com/en-us/azure/key-vault/keys/about-keys-details#ec-algorithms)) when we do the work to add support there.